azure-servicebus/pom.xml

```diff
@@ -96,5 +91,10 @@
 			 <artifactId>async-http-client</artifactId>
 			 <version>2.5.2</version>
 		 </dependency>
+		 <dependency>
+		    <groupId>com.google.code.gson</groupId>
+		    <artifactId>gson</artifactId>
+		    <version>2.8.5</version>
+		</dependency>
```

Comment:
GSON was included in ADAL, so after that's removed this needs to be added explictly now.